fasttext/debian/rules

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/make -f
+#export DH_VERBOSE = 1
+
+%:
+	dh $@ --buildsystem=cmake
```

Comment:
Is `--buildsystem=cmake` needed?

Reply:
It is needed because fastText is intended to use with cmake, debhelper can't handle it without --buildsystem=cmake correctly.

Reply:
`debhelper` can't detect `CMakeLists.txt`?

Reply:
Yes. In this case, if --buildsystem=cmake is omitted, Makefile used but there is no install target.

Reply:
I see.